libs/npm/core/src/hooks/api/interfaces/IContentModel.ts

```diff
@@ -48,20 +48,20 @@ export interface IContentModel extends IAuditColumnsModel {
   postedOn?: string;
   publishedOn: string;
   summary: string;
-  body?: string;
+  body: string;
```

Comment:
These interfaces represent the data models from the API.  They should be typed accordingly.